exonum/src/runtime/error.rs

```diff
@@ -74,25 +74,24 @@ impl ErrorKind {
         ErrorKind::Service { code: code.into() }
     }
 
-    fn into_raw(self) -> (u8, u8) {
+    fn into_raw(self) -> (runtime::ErrorKind, u8) {
```

Comment:
Maybe change it to `runtime_proto` to explicitly mark that it's not the `runtime` mod in which `error.rs` lies in?